### DISCUSSION TITLE
Maps vs. Overlays

### DISCUSSION THREAD

**Discussion by Heineken** [bailuk/AAT#151] [Ideas]
I find the map selection confusing, as there are actually only two map providers: MapsForge (not working for me right now, btw.) and Mapnik. The other options are overlays in the way I understand it.
Under "Overlays" I find 4 zeros (4 slots?) and no hint what they mean. (I assume I could use a recorded track here. Right?)
Suggestion:
1. Move all "maps" but the two actual maps to overlays.
2. Show a message when selecting an empty overlay slot.
Or, even better, give an option to select a track.

**Comment by RealVegOs** [comment]
I use overlays for routes. I put them directly in the overlay folder with a file manager. In my opinion, this is not the place for maps.
